### PR TITLE
[skip ci] Add ActiveRecord::Base.normalizes to 7.1 release notes

### PR DESCRIPTION
**Pull request by deepakmahakale**
### Motivation / Background

This Pull Request has been created because [REPLACE ME]

### Detail

This Pull Request changes [REPLACE ME]

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [ ] Tests are added or updated if you fix a bug or add a feature.
* [ ] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

**Comment by deepakmahakale** [comment]
@rafaelfranca do let me know if this is good or if we need to add more / reduce the description.

This is the same content from changelog

File: guides/source/7_1_release_notes.md [hunk 1]
```diff
@@ -58,7 +58,31 @@ getting your Rails application up and running in a production environment.
 
 ### Add `ActiveRecord::Base.normalizes`
 
-TODO: Add description https://github.com/rails/rails/pull/43945
+A normalization is applied when the attribute is assigned or updated,
+and the normalized value will be persisted to the database.
+The normalization is also applied to the corresponding keyword argument of finder methods.
+This allows a record to be created and later queried using unnormalized values.
```
Comment (adrianna-chang-shopify) [inline guides/source/7_1_release_notes.md:64]:
```suggestion
Normalizations can be declared for attribute values. The normalization
takes place when the attribute is assigned or updated, and will be persisted to the database.
Normalization is also applied to corresponding keyword arguments in finder methods,
allowing records to be queried using unnormalized values.
```

Small suggestion, what do you think?

Comment (deepakmahakale) [reply guides/source/7_1_release_notes.md:64]: Looks good.

**Comment by adrianna-chang-shopify** [comment]
@deepakmahakale can you squash your commits and then I'll merge? Thanks!

**Comment by deepakmahakale** [comment]
Done

**Pull request merged**